templates/Pages/home.php

```diff
@@ -174,6 +174,14 @@
                     </div>
                 </div>
                 <hr>
+                <div class="row">
+                    <div class="column links">
+                        <h3>Demo Controllers</h3>
+                        <a href="/posts">Posts Controller Index</a>
```

Comment:
Please use the HtmlHelper to generate the links. These won't work if someone installs the app in a sub directory.